### ISSUE TITLE
Stopped working with Home Assistant Core 2022.3.0

### ISSUE THREAD

**Issue by BeanXOfficial** [fishbigger/HomeAssistant-Tapo-P100-Control#54]
I haven't changed anything in my config and Home Assistant gives me the following error: 

`Platform error switch.tapo_p100_control - Requirements for tapo_p100_control not found: ['PyP100==0.0.18'].
Platform error switch.tapo_p100_control - Requirements for tapo_p100_control not found: ['PyP100==0.0.18'].
Platform error switch.tapo_p100_control - Requirements for tapo_p100_control not found: ['PyP100==0.0.18'].
Platform error switch.tapo_p100_control - Requirements for tapo_p100_control not found: ['PyP100==0.0.18'].`

Here is my config (in a seperate switches.yaml):

```
- platform: tapo_p100_control
  ip_address: 192.168.178.48
  email: !secret email
  password: !secret tapo_password
- platform: tapo_p100_control
  ip_address: 192.168.178.47
  email: !secret email
  password: !secret tapo_password
- platform: tapo_p100_control
  ip_address: 192.168.178.37
  email: !secret email
  password: !secret tapo_password
- platform: tapo_p100_control
  ip_address: 192.168.178.36
  email: !secret email
  password: !secret tapo_password
```

**Comment by programmer24601** [comment]
Also stopped working for me, but the homeassistant logs indicate:

```
Unable to install package PyP100==0.0.18: ERROR: Cannot install pyp100==0.0.18 because these package versions have conflicting dependencies. ERROR: ResolutionImpossible: for help visit 
```
Rolling back to home-assistant 2022.2.9 resolves the issue.

**Comment by jtizzel** [comment]
I also have this issue, and have had to roll back to previous Core version. 
I’ve attempted the upgrade twice.

**Comment by RudolfvGanz** [comment]
Similar error here after updating:
Requirements for tapo_p100_control not found: ['PyP100==0.0.11'].

**Comment by pascalprey** [comment]
Same problem here. After update to 2022.3.X the integration doesn't work anymore.

I switched to the following integration:
https://github.com/petretiandrea/home-assistant-tapo-p100

The integration can be installed via HACS. Configuration is very easy. 

Maybe it's a workaround for you, too.

**Comment by jtizzel** [comment]
Yes I also did this. Thanks.

**Comment by kerozoli** [comment]
Strange I tested that repo with 2022.3.X and it didn't work for me with 2022.3.1 :
https://github.com/petretiandrea/home-assistant-tapo-p100/issues/148
@pascalprey Your tapo devices work with that repository with 2022.3.1 ?
Thanks

**Comment by pascalprey** [comment]
@kerozoli yes, I have four P100 WiFi plugs and every plug works perfectly with the integration from above. No issues.

**Comment by lanerobertlane** [comment]
Same issue for me. Tried a work around of changing the mainfest.json to `"PyP100>=0.0.11"` to see if that would make it happy, and knowing that was one of the previous versions I used. It didn't work and changed the error from 
`Unable to install package PyP100==0.0.18: ERROR: Cannot install pyp100==0.0.18 because these package versions have conflicting dependencies. ERROR: ResolutionImpossible: for help visit` 
to 
`Unable to install package PyP100>=0.0.11: ERROR: Cannot install pyp100>=0.0.11 because these package versions have conflicting dependencies. ERROR: ResolutionImpossible: for help visit`

tried a `pip3 install PyP100` through sshing into my HA container and it failed too, something to do with `pycryptodome`. Was going to try to `--nodeps` but didn't wanna risk it. 

Rolled back to 2021.12 for now.

**Comment by ajernejcic** [comment]
same here;  pycryptodome==3.9.8 has to be built, but no gcc in the container.

**Comment by vascofazza** [comment]
Same issue here, any workaround?

**Comment by ajernejcic** [comment]
Hi, i switched to https://github.com/petretiandrea/home-assistant-tapo-p100, installed via hacs and add devices via integration. had to reconfigure some schedules and scenes. but vi is my friend ;)

**Comment by RudolfvGanz** [comment]
> Similar error here after updating: Requirements for tapo_p100_control not found: ['PyP100==0.0.11'].

Followed the advice and switched to https://github.com/petretiandrea/home-assistant-tapo-p100. Installed manually in custom_components, and added through the UI - automations are again working

**Comment by lanerobertlane** [comment]
FYI, for anyone who want's to try the petretiandrea link being shared here, the issue persists if you're using the docker version of HA, although their appears to be an issue for it actively being discussed. https://github.com/petretiandrea/home-assistant-tapo-p100/issues/153

**Comment by fishbigger** [comment]
Hi,
Sorry this has taken so long. I believe the issue is to do with outdated dependencies within PyP100. I have now updated them and will produce a fixed version of this soon.

**Comment by fishbigger** [comment]
Fixed in [257d414](https://github.com/fishbigger/HomeAssistant-Tapo-P100-Control/commit/257d4146d87b144924b1ce0fcec28d4933df4fa2)
